cmake/examples.cmake

```diff
@@ -2,7 +2,7 @@ FIND_PACKAGE(OpenCV)
 FIND_PACKAGE(gflags)
 FIND_PACKAGE(Threads REQUIRED)
 
-ADD_LIBRARY(helpers examples/input.cpp examples/vis.cpp)
+ADD_LIBRARY(helpers examples/input.cpp examples/vis.cpp examples/thread_pool.cpp examples/spinlock.cpp)
```

Comment:
spinlock.cpp is no longer used.

Reply:
Yes. But I still want to keep it as a possible concurrent primitive in the future. Of course, I can remove that.